app.js

```diff
@@ -9,6 +9,7 @@ const SequelizeStore = require('connect-session-sequelize')(session.Store);
 const indexRouter = require('./routes/index');
 const storiesRouter = require('./routes/stories');
 const usersRouter = require('./routes/users');
+const apiRoutes = require('./routes/api/')
```

Comment:
The router is imported here, but there is no call to `app.use` in the code with the imported `apiRoutes`